### PR TITLE
add base ussd codes for some banks to the JSON

### PR DESCRIPTION
**Pull request by tomiadebanjo**
Nice work. I added the base USSD codes for the banks that have USSD code

### CODE REVIEW

File: banks.js [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
-  { name: 'STANBIC IBTC BANK LTD', code: '221', slug: 'SIB' },
-  { name: 'STANDARD CHARTERED BANK NIGERIA LTD', code: '068', slug: 'SCB' },
-  { name: 'STERLING BANK PLC', code: '232', slug: 'STB' },
-  { name: 'SUNTRUST BANK NIGERIA LTD', code: '100', slug: 'SBN' },
-  { name: 'UNION BANK OF NIGERIA PLC', code: '032', slug: 'UBN'},
-  { name: 'UNITED BANK FOR AFRICA PLC', code: '033', slug: 'UBA' },
-  { name: 'UNITY BANK PLC', code: '215', slug: 'UNB' },
-  { name: 'WEMA BANK PLC', code: '035', slug: 'WEM' },
-  { name: 'ZENITH BANK PLC', code: '057', slug: 'ZIB' }
+	{ name: 'ACCESS BANK PLC', code: '044', slug: 'ACC', 'ussd code': '*901#' },
```
[the remark this answers is not in the document]
Comment (tomiadebanjo) [reply banks.js]: > a better readable key name is `ussdCode` or `ussd_code`. Can you change it to that?

I changed the key name to `ussdCode`

[the remark this answers is not in the document]
Comment (tomiadebanjo) [reply banks.js]: > Also, I think your editor added some padding to the object indentation structure, was that intentional?

It wasn't intentional. My editor added the extra padding

[the remark this answers is not in the document]
Comment (BolajiOlajide) [reply banks.js]: Oh, nice. Thanks for clarifying.

File: banks.js [hunk 2]
```diff
@@ -1,25 +1,25 @@
...
+  { name: 'ECOBANK NIGERIA PLC', code: '050', slug: 'EBN', 'ussd code': '*326#' },
+  { name: 'ENTERPRISE BANK', code: '084', slug: 'EPB', 'ussd code': '*901#' },
+  { name: 'FIDELITY BANK PLC', code: '070', slug: 'FDB', 'ussd code': '*770#' },
+  { name: 'FIRST BANK NIGERIA LIMITED', code: '011', slug: 'FBN', 'ussd code': '*894#' },
+  { name: 'FIRST CITY MONUMENT BANK PLC', code: '214', slug: 'FCB', 'ussd code': '*329#' },
+  { name: 'GUARANTY TRUST BANK PLC', code: '058', slug: 'GTB', 'ussd code': '*737#' },
+  { name: 'HERITAGE BANK PLC', code: '030', slug: 'HTB', 'ussd code': '*322#' },
+  { name: 'KEY STONE BANK', code: '082', slug: 'KSB', 'ussd code': '*533#' },
+	{ name: 'MAINSTREET BANK', code: '014', slug: 'MSB' },
+	{ name: 'POLARIS BANK LIMITED', code: '076', slug: 'PLB' },
```
[the remark this answers is not in the document]
Comment (BolajiOlajide) [reply banks.js]: I'm looking through.

**Pull request merged**